### PR TITLE
Release version 1.0.0

### PR DESCRIPTION
**Pull request by robinpokorny**
I added a changelog, version badge, and my name to contributors.

Please create tag and publish to NPM.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## 1.O.0 - 2016-04-14
```
Comment (agudulin) [inline CHANGELOG.md]:
``` diff
- ## 1.O.0 - 2016-04-14
+ ## 1.0.0 - 2016-04-14
```

File: package.json [hunk 2]
```diff
@@ -41,5 +41,8 @@
     "postpublish": "rimraf index.es5.js"
   },
   "author": "Alexander Gudulin <alexandr.gudulin@gmail.com> (http://gudulin.com)",
+  "contributors": [
+    "Robin Pokorny <me@robinpokorny.com> (http://robinpokorny.com)"
```
Comment (agudulin) [inline package.json]: add paranoida too, please :)

**Comment by robinpokorny** [comment]
Updated :)

**Comment by agudulin** [comment]
Perfect :+1:

**Pull request merged**